test/smoke/testApps/PerfTestAppSpringBoot/build.gradle

```diff
@@ -0,0 +1,49 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'org.springframework.boot:spring-boot-gradle-plugin:1.5.19.RELEASE'
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'eclipse'
+apply plugin: 'war'
+apply plugin: 'org.springframework.boot'
+
+compileJava.sourceCompatibility = 1.7
+compileJava.targetCompatibility = 1.7
+compileSmokeTestJava.sourceCompatibility = 1.8
+compileSmokeTestJava.targetCompatibility = 1.8
+
+dependencies {
+    compile springBootStarterJar
+    compile group: 'org.springframework.boot', name: 'spring-boot-starter-web'
+    compile group: 'org.springframework.boot', name: 'spring-boot-starter-tomcat'
+    compile 'com.google.guava:guava:20.0'
+    compile 'org.apache.httpcomponents:httpclient:4.5.3'
+
+    compileOnly 'org.apache.tomcat.embed:tomcat-embed-jasper'
```

Comment:
why do we need embedded tomcat? We are deploying it in tomcat container I believe, so not needed unless you have some reason.

Reply:
for jsp compilation

Reply:
Got it. Thanks